robotiq_driver/CMakeLists.txt

```diff
@@ -1,12 +1,15 @@
+# See https://docs.ros.org/en/humble/How-To-Guides/Ament-CMake-Documentation.html
+
 cmake_minimum_required(VERSION 3.8)
-project(robotiq_driver)
+project(robotiq_driver VERSION 0.1.0 LANGUAGES CXX)
```

Comment:
I do not think we should not be setting the `robotiq_driver VERSION` here it is set in the package.xml and the ros release infrastrure and process will bump it accordingly.

Reply:
I think this is an outdated comment. I can see the version and the language set.

Reply:
![image](https://github.com/PickNikRobotics/ros2_robotiq_gripper/assets/55676/2702a34e-a53f-4688-9ee6-96091097aef4)

I also see the version and the language being set, but I do not think you should be setting the version here.

```suggestion
project(robotiq_driver LANGUAGES CXX)
```

ament_cmake gets information from `package.xml` where the version is being set to 0.0.1 and the `ament_package()` call at the end of this CMake file will bump the version accordingly when `catkin_prepare_release` and `bloom` are run to release the package... when releasing with ROS you shouldn't bump the version number you should let the ROS infrastructure hand that... the default of catkin_prepare_release is to `--bump patch` but it can be called with `--bump minor` or `--bump major`